cmd/run/run.go

```diff
@@ -188,6 +189,22 @@ func isPipe(r io.Reader) bool {
 	return false
 }
 
+// promptFile mirrors the format of .prompt.yml
+type promptFile struct {
+	Name            string `yaml:"name"`
```

Comment:
Is there a way with YAML parsing in Go to specify that fields are optional? Maybe they all default to optional and you have to flag them to make them required. 🤔

Reply:
It's optional by default.